LongWarOfTheChosen/Config/XComClassData.ini

```diff
@@ -1613,3 +1613,246 @@ bHasClassMovie=true
 +bHideInCharacterPool=true
 +IconImage="img:///UILibrary_StrategyImages.MissionIcon_Resistance"
 
+
+[Skirmisher X2SoldierClassTemplate]
+!RandomAbilityDecks=()
++RandomAbilityDecks=(DeckName="SkirmisherTier1XComAbilities", \\
+					Abilities=((AbilityName="LightningReflexes_LW"), \\
+							   (AbilityName="SkirmisherReturnFire", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon), \\
+							   \\
+							   (AbilityType=(AbilityName="SkirmisherInterrupt")), \\
+							   (AbilityName="NeedleGrenades"), \\
+							  ))
++RandomAbilityDecks=(DeckName="SkirmisherTier2XComAbilities", \\
+					Abilities=((AbilityName="HeavyOrdnance_LW"), \\
+							   (AbilityName="CombatPresence"), \\
+							   (AbilityName="HeavyOrdnance_LW"), \\
+							   (AbilityName="Sapper"), \\
+							   \\
+							   (AbilityType=(AbilityName="LoneWolf")), \\
+							   (AbilityType=(AbilityName="TotalCombat")), \\
+							   (AbilityType=(AbilityName="ZeroIn", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon)), \\
+							  ))
++RandomAbilityDecks=(DeckName="SkirmisherTier3XComAbilities", \\
+					Abilities=((AbilityName="BluescreenBombs"), \\
+							   (AbilityName="HEATWarheads"), \\
+							   (AbilityName="BiggestBooms_LW"), \\
+							   \\
+							   (AbilityName="VolatileMix"), \\
+							   (AbilityName="StingGrenades", ApplyToWeaponSlot=eInvSlot_Utility), \\
+							   (AbilityName="Bombard_LW"), \\
+							   (AbilityName="FullKit"), \\
+							  ))
+!SoldierRanks=()  
+; squaddie
++SoldierRanks=(AbilitySlots=((AbilityType=(AbilityName="Battlemaster", ApplyToWeaponSlot=eInvSlot_PrimaryWeapon)), \\
+							(AbilityType=(AbilityName="Justice")), \\
+							(AbilityType=(AbilityName="Tradecraft")), \\
```

Comment:
Actually, it's basically the same as giving a soldier an SMG. Which makes me wonder, why not reduce the covertness of the Bullpup instead? In fact, I think I'd rather have a reduced covertness on Bullpup and add Tradecraft as a later perk.

Not really sure why the Skirmisher also has suppression and a grenade slot. Seems a bit over the top for Squaddie. I would at least drop suppression.

Reply:
Well as far as convert a 5 man squad to a 6 man squad, you are only going to be doing this when you have a skirmisher on the squad so it's not like you will be able to do this for every mission. Also tradecraft fits them more thematically as well because they were former advent so it would make sense that they would be able to infiltrate missions better with the knowledge they have of the advent network.

As far as suppression goes, I agree that they probably do not really need it. I never used it much but can also see it being a bit too strong at squaddie. Will give that the axe. The grenade slot on the other hand we will keep, the skirmisher random decks have a lot of grenade skills we would like them to keep.

Reply:
So we just wen with removing suppression and tradecraft here. What was done is added battlemaster to infiltration settings and gave it the same amount of reduced infiltration time as tradecraft. Battlemaster now has an effect that is basically giving it the damn good ground bonuses.